Header Header

int32 distance
int32 theta
string object_type